### ISSUE TITLE
Incomplete Applications - Transferring from RegAdmin to BriReg

### ISSUE THREAD

**Issue by Shorashim** [MarchingLizard/brifony#30]
Over the weekend, I noticed that some of the applications are not transferring over from regadmin to brireg. When looking at the numbers, we have 219 in the brireg query and 733 in the regadmin query. I know that this is partially because we cancelled applications of those who are ineligible or interested in different seasons, but I imagine that this does not account for the entire 514 difference.

I found an application that is incomplete, but not in the query to focus on as an example. Hannah Taub ( ID: 702228433149) has not transferred over to brireg, but Aaron Abrams (ID: 196866741948) has. From looking over the applications, it is clear that Hannah has much less information filled out than Aaron, but since they both have IDs, I was under the impression that they should both be able to show up in the query in brireg. Is that correct?  Please let me know if there is something I am missing and if there are next steps we can take to ensure we are not missing out on outreach.

Thank you!

**Comment by IshaDakota** [comment]
@Shorashim Yes. This is indeed a bug introduced when we pulled out the declined applicants. Will fix.

**Comment by Shorashim** [comment]
Thanks, Adam! Please let us know when it is fixed, so I can make sure to reach out to outstanding applicants. If you need anything else on our end, please let me know!

**Comment by IshaDakota** [comment]
now fixed.

**Comment by Shorashim** [comment]
The number in brireg for incompletes just jumped from 210 to 910. Since there are only 736 in the regadmin query and a number that have been deleted, I think there is still some sort of error. I am looking into this, but since you just fixed it, any idea what can be done to rectify this? I am thinking that the amount should be somewhere in the 400s/500s. Thank you!

**Comment by IshaDakota** [comment]
I'm not sure where BRI gets that number, but what you are seeing is the exact number of applications marked as incomplete. The only difference in numbers is because we keep applicants in our system who transfer out. But even if you remove these, it's only about 30-40 applicants.

If you play with the report generator below the numbers, you can see that id you set the "completion status" to incomplete, but set the other options to different variables, you could get anywhere from 400 - 1300 applicants.
